pkgs/applications/networking/browsers/w3m/default.nix

```diff
@@ -19,6 +19,8 @@ stdenv.mkDerivation rec {
     sha256 = "1qx9f0kprf92r1wxl3sacykla0g04qsi0idypzz24b7xy9ix5579";
   };
 
+  NIX_LDFLAGS = "-lsocket -lnsl";
```

Comment:
Yes please, at least it makes it clear it's only a sunos issue, for those that will work on that derivation.